examples/demo-rollup/tests/evm/evm_logs.rs

```diff
@@ -1,373 +1,46 @@
-use std::ops::Range;
```

Comment:
The big diff is a consequence of file renaming. The only relevant part is `evm_test_get_logs`